dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/CDIConfigByACPTests.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2015, 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ */
+package com.ibm.ws.jsf22.fat.tests;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.jsf22.fat.CDITestBase;
+
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * This is one of four CDI test applications, with configuration loaded in the following manner:
+ * CDIConfigByACP - Application Configuration Populator loading of the class files
+ *
+ * We're extending CDITestBase, which has common test code which is found under test-applications/CDICommon.
+ *
+ * NOTE: These tests should not run with jsf-2.3 feature because constructor injection is not supported.
+ * As a result, these tests were modified to run in the JSF 2.3 FAT bucket without constructor injection.
+ */
+@Mode(TestMode.FULL)
+@SkipForRepeat("JSF-2.3")
+@MinimumJavaLevel(javaLevel = 7)
```

Comment:
To the best of my knowledge we don't need to specify the MinimumJavaLevel at the class level if you're already specifying it in the bnd.bnd : https://github.com/OpenLiberty/open-liberty/wiki/FAT-tests#running-fat-tests-locally-with-different-java-levels